### PR TITLE
 April updates to Management API Docs

### PR DESCRIPTION
**Pull request by OSIErickRC**
- New information for Namespace, Role, & Tenant routes
- Added new route to Service Blog

### CODE REVIEW

File: Documentation/Management/Account_Namespace.md [hunk 1]
```diff
@@ -136,7 +136,7 @@ Creates a new `Namespace` in the specified `Tenant`.
 
 ### Http
 
-`POST api/v1-preview/Tenants/{tenantId}/Namespaces`
+`POST api/v1-preview/Tenants/{tenantId}/Namespaces/{namespaceId?}`
```
Comment (osimarkrogers) [inline Documentation/Management/Account_Namespace.md]: Question mark in POST statement is intentional?

File: Documentation/Management/Account_ServiceBlog.md [hunk 2]
```diff
@@ -47,3 +47,36 @@ Any identity, including anonymous, can retrieve service blog entries.
...
+
+### Security
+
+Any identity, including anonymous, can retrieve service blog entries.
+
+### Returns
+
+| Status Code | Return Type | Description | 
+ | --- | --- | ---  | 
+| 200 | SdsServiceBlogEntry | Returns an response object including the specified blog entry. |
```
Comment (osimarkrogers) [inline Documentation/Management/Account_ServiceBlog.md]: Should read "Returns a response object...."

File: Documentation/Management/Account_Namespace.md (hunk 1 above)
Comment (OSIErickRC) [reply Documentation/Management/Account_Namespace.md]: It's an optional parameter.

File: Documentation/Management/Account_ServiceBlog.md (hunk 2 above)
Comment (OSIErickRC) [reply Documentation/Management/Account_ServiceBlog.md]: Fixed.

File: Documentation/Management/Account_Namespace.md (hunk 1 above)
Comment (osimarkrogers) [reply Documentation/Management/Account_Namespace.md]: Understood.  Merging to master.

**Pull request merged**